main.cc

```diff
@@ -68,6 +68,7 @@
 #include "gms/feature_service.hh"
```

Comment:
The prefix for the commit subject should be "transport:" rather than "db:" since it applies to the transport layer.

Reply:
done